lib/truncate-collections/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+class TruncateCollection {
+
+  constructor(mongoose) {
+    this._mongoose = mongoose;
+    this.truncate = this.truncate.bind(this);
+  }
+
+
+  truncate() {
+    let truncateCollections = this._collectionNames
+      .map((collectionName) => this._collections[collectionName])
```

Comment:
you should delete the brackets for single input parameter (linter should throw an error about this)